website/content/docs/ecs/manual/acl-controller.mdx

```diff
@@ -0,0 +1,213 @@
+its own instance of the ACL controller, and each instance of the ACL controller should have a unique
+name prefix.
+
+## Task Definition
+
+You must create a task definition to deploy the ACL controller in your ECS cluster.
+The ACL controller must run in the same ECS cluster hosting your service mesh application
+tasks.
+
+The following example shows how the task definition should be configured for the ACL controller.
+
+```json
+{
+  "family": "my-consul-acl-controller".
+  "networkMode": "awsvpc",
+  "containerDefinitions": [
+    {
+      "name": "acl-controller",
+      "image": "public.ecr.aws/hashicorp/consul-ecs:<CONSUL_ECS_VERSION>",
+      "essential": true,
+      "command": [
+        "acl-controller",
+        "-consul-client-secret-arn", "arn:aws:secretsmanager:us-west-2:000000000000:secret:<PREFIX>-consul-client-token",
+        "-secret-name-prefix", "<PREFIX>",
+      ],
+      "secrets": [
+        {
+          "name": "CONSUL_HTTP_TOKEN",
+          "valueFrom": "arn:aws:secretsmanager:us-west-2:000000000000:secret:my-consul-bootstrap-token"
+        },
+        {
+          "name": "CONSUL_CACERT_PEM",
+          "valueFrom": "arn:aws:secretsmanager:us-west-2:000000000000:secret:my-consul-ca-cert"
+        }
+      ],
+      "environment": [
+        {
+          "name": "CONSUL_HTTP_ADDR",
+          "value": "<Consul server HTTP API address>"
+        }
+      ]
+    }
+  ]
+}
+```
+
+You must include the following top-level fields.
+
+| Field name    | Type    | Description                                                                  |
+| -----------   | ------- | ---------------------------------------------------------------------------- |
+| `family`      | string  | The task family name of your choice.                                         |
+| `networkMode` | string  | Must be `awsvpc`, which is the only network mode supported by Consul on ECS. |
+
+In the `containerDefinitions` list, include one container with the following fields.
+
+| Field name    | Type    | Description                                                                                                                      |
+| -----------   | ------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `name`        | string  | The container name, which should be `acl-controller`                                                                             |
+| `image`       | string  | The `consul-ecs` image. Use our public AWS registry, `public.ecr.aws/hashicorp/consul-ecs`, to avoid rate limits.                |
+| `command`     | list    | Must be set as shown. The startup command for the ACL controller.                                                                |
+| `essential`   | boolean | Must be `true` to ensure the container health ties into the health of the task.                                                  |
```

Comment:
```suggestion
| `essential`   | boolean | Must be `true` to ensure the container health ties into the health of the task.                                                  |
```
What do we mean by "ties into the health task"? Would "is associated with the health of the task" be clearer?

Reply:
> Would "is associated with the health of the task"

Yeah, or maybe "effects the health of the task"?

The specific behavior in the [AWS docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_healthcheck) is: 

> The following describes the possible `healthStatus` values for a task. The container health check status of nonessential containers do not have an effect on the health status of a task.
>
> * `HEALTHY`—All essential containers within the task have passed their health checks.
> * `UNHEALTHY`—One or more essential containers have failed their health check.
> * `UNKNOWN`—The essential containers within the task are still having their health checks evaluated or there are no container health checks defined.

Reply:
I went with the following:

> Must be `true` to ensure the health of your application container effects the health status of the task.

Reply:
And corrected to `affects` (verb) instead of `effects`